poc/vidpf.py

```diff
@@ -155,7 +155,14 @@ def eval(cls, agg_id, correction_words, init_seed, level, prefixes, cs_proofs, p
         for prefix in prefixes:
             (_seed, _ctrl, y, _pi_proof) = prefix_tree_share[(prefix, level)]
             out_share.append(y if agg_id == 0 else vec_neg(y))
-        return (out_share, pi_proof + path_proof)
+
+        # Compute the Aggregator's share of `beta`.
+        y0 = prefix_tree_share[(0, 0)][2]
+        y1 = prefix_tree_share[(1, 0)][2]
+        beta_share = vec_add(y0, y1)
+        if agg_id == 1:
+            beta_share = vec_neg(beta_share)
```

Comment:
Just to follow up on the issue from the call: We need to negate aggregator 1's share; I was negating 0's.

Reply:
Yup, saw that :)